src/js/common/index.js

```diff
@@ -5,9 +5,6 @@
 // other such libraries in here. Most of the site does not use these legacy
 // frameworks and it belongs in a lower-level module.
 
-// Global site style.
-require('../../sass/style.scss');
```

Comment:
@webinista I'm finally getting to merging this PR, and I think this small change of moving the import of `style.scss` to `no-react-entry.js` is fixing the problem of the USWDS being imported twice. Does this change look correct to you?